code_samples/user_management/oauth_google/config/services.yaml

```diff
@@ -0,0 +1,43 @@
+# This file is the entry point to configure your own services.
+# Files in the packages/ subdirectory configure your dependencies.
+
+# Put parameters here that don't need to change on each machine where the app is deployed
+# https://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
+parameters:
+    google_recaptcha_site_key: '%env(GOOGLE_RECAPTCHA_SITE_KEY)%'
+
+services:
+    # default configuration for services in *this* file
+    _defaults:
+        autowire: true      # Automatically injects dependencies in your services.
+        autoconfigure: true # Automatically registers your services as commands, event subscribers, etc.
+
+    # makes classes in src/ available to be used as services
+    # this creates a service per class whose id is the fully-qualified class name
+    App\:
+        resource: '../src/'
+        exclude:
+            - '../src/DependencyInjection/'
+            - '../src/Entity/'
+            - '../src/Kernel.php'
+            - '../src/Tests/'
+
+    # controllers are imported separately to make sure services can be injected
+    # as action arguments even if you don't extend any base controller class
+    App\Controller\:
+        resource: '../src/Controller/'
+        tags: ['controller.service_arguments']
+
+    # add more service definitions when explicit configuration is needed
+    # please note that last definitions always *replace* previous ones
+
+    App\OAuth\GoogleResourceOwnerMapper:
```

Comment:
I think we need only lines from 34 to the end and actually, it most likely should be moved to a separated file, WDYT @adamwojs?

Reply:
Only lines 34+ will be embedded in the doc, see here: https://github.com/ezsystems/developer-documentation/pull/1304/files#diff-8fcf856c9fcd93fd5bbc9d15cf7c49a16463d30067a431d946618eb1d4c0bbd0R51
The full file is included for testing purposes, but not rendered.

Reply:
Oh, sorry, I didn't notice it. All good 👍